spec/lib/triannon-client/triannon_client_spec.rb

```diff
@@ -0,0 +1,315 @@
+      expect(const).to include('http://www.w3.org/ns/oa-context-20130208.json')
+    end
+    it 'CONTENT_TYPE_IIIF' do
+      const = TriannonClient::TriannonClient::CONTENT_TYPE_IIIF
+      expect(const).to be_instance_of String
+      expect(const).to include('application/ld+json')
+      expect(const).to include('http://iiif.io/api/presentation/2/context.json')
+    end
+    it 'CONTENT_TYPE_OA' do
+      const = TriannonClient::TriannonClient::CONTENT_TYPE_OA
+      expect(const).to be_instance_of String
+      expect(const).to include('application/ld+json')
+      expect(const).to include('http://www.w3.org/ns/oa-context-20130208.json')
+    end
+    it 'JSONLD_TYPE' do
+      const = TriannonClient::TriannonClient::JSONLD_TYPE
+      expect(const).to be_instance_of String
+      expect(const).to eql('application/ld+json')
+    end
+  end
+
+  describe 'responds to' do
+    before(:example) do
+      @tc = TriannonClient::TriannonClient.new
+    end
+    it 'delete_annotation' do
+      expect(@tc).to respond_to(:delete_annotation)
+    end
+    it 'get_annotations' do
+      expect(@tc).to respond_to(:get_annotations)
+    end
+    it 'get_annotation' do
+      expect(@tc).to respond_to(:get_annotation)
+    end
+    it 'get_iiif_annotation' do
+      expect(@tc).to respond_to(:get_iiif_annotation)
+    end
+    it 'get_oa_annotation' do
+      expect(@tc).to respond_to(:get_oa_annotation)
+    end
+    it 'site' do
+      expect(@tc).to respond_to(:site)
+    end
+    # utilities
+    it 'response2graph' do
+      expect(@tc).to respond_to(:response2graph)
+    end
+    it 'annotation_id' do
+      expect(@tc).to respond_to(:annotation_id)
+    end
+    it 'annotation_uri' do
+      expect(@tc).to respond_to(:annotation_uri)
+    end
+  end
+
+  describe 'has private method' do
+    before(:example) do
+      @tc = TriannonClient::TriannonClient.new
+    end
+    it 'check_id' do
+      expect(@tc).not_to respond_to(:check_id)
+    end
+    it 'check_content_type' do
+      expect(@tc).not_to respond_to(:check_content_type)
+    end
+  end
+
+  describe "#delete_annotation" do
+    before(:example) do
+      @tc = TriannonClient::TriannonClient.new
+    end
+    def test_delete_for_response_code(status, response)
+      allow_any_instance_of(RestClient::Response).to receive(:code).and_return(status)
+      expect(@tc.delete_annotation('anything_here_is_OK')).to be response
+    end
+    it "returns false for a 500 response to a DELETE request" do
+      test_delete_for_response_code(500, false)
+    end
+    it "returns true for a 200 response to a DELETE request" do
+      test_delete_for_response_code(200, true)
+    end
+    it "returns true for a 202 response to a DELETE request" do
+      test_delete_for_response_code(202, true)
+    end
+    it "returns true for a 204 response to a DELETE request" do
+      test_delete_for_response_code(204, true)
+    end
+    it 'checks the annotation ID' do
```

Comment:
this is fine.  I would probably change to  "calls check_id", but it's a quibble.